java-backend/src/main/java/com/custom/todo/list/app/controller/TaskHttp.java

```diff
@@ -0,0 +1,75 @@
+package com.custom.todo.list.app.controller;
+
+import com.custom.todo.list.app.dto.AddTaskRequest;
+import com.custom.todo.list.app.dto.UpdateTaskRequest;
+import com.custom.todo.list.app.service.AddTaskService;
+import com.custom.todo.list.app.service.DeleteTaskService;
+import com.custom.todo.list.app.service.ListTasksService;
+import com.custom.todo.list.app.service.UpdateTaskService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Validated
+@RestController
+@RequestMapping(path = "/v1.0/api/todo")
```

Comment:
Maybe we can remove the versioning here so no changes are required in the frontend?
```suggestion
@RequestMapping(path = "/api/todo")
```

Reply:
i added it as a good practice, to give a good starting point to guyss, what do y think?